packages/fetchai/connections/p2p_libp2p/utils/utils.go

```diff
@@ -231,6 +245,12 @@ func BTCPubKeyFromFetchAIPublicKey(publicKey string) (*btcec.PublicKey, error) {
 	return pbk, err
 }
 
+// BTCPubKeyFromEthereumPublicKey create libp2p public key from ethereum uncompressed
+//  hex encoded secp256k1 key
+func BTCPubKeyFromEthereumPublicKey(publicKey string) (*btcec.PublicKey, error) {
+	return BTCPubKeyFromUncompressedHex(publicKey[2:])
```

Comment:
so dropping `0x` I imagine? Worth adding as a comment probably.

Reply:
I don't think we need it, should be transparent